controls.py

```diff
@@ -0,0 +1,80 @@
+import pygame
+import sys
+
+
+class Controls:
+    def __init__(self, comms, exit_program):
```

Comment:
Consider statically typing your parameters (since we are using Python 3.9+)
```python
def __init__(self, comms: Comms,  exit_program: Exit_Program):
```